src/library/autodj/autodjprocessor.cpp

```diff
@@ -1046,7 +1061,7 @@ void AutoDJProcessor::calculateTransition(DeckAttributes* pFromDeck,
 
     // We require ADJ_IDLE to prevent changing the thresholds in the middle of a
     // fade.
-    if (m_eState != ADJ_IDLE) {
+    VERIFY_OR_DEBUG_ASSERT(m_eState == ADJ_IDLE) {
```

Comment:
I am confused why you changed this to a debug assertion. There is now code scattered throughout AutoDJProcessor repeating this check before calling calculateTransition. Why is it not sufficient to check this here at the top of calculateTransition? Aren't those other checks redundant?

Reply:
I have moved the check outside to clearly mark all calling functions as "guard for manual user changes only".
I had a knot in my head when debugging because of all these slots. 
This change helped me finally for a better understanding. I have kept the change, because it also saved some cpu cycles.